HACKING.md

```diff
@@ -0,0 +1,99 @@
+# Building barnabas
+
+Barnabas is using `make` as its main build system. Our make build supports 
+several different targets mainly for building and pushing docker images.
+
+<!-- TOC depthFrom:2 -->
+
+- [Docker images](#docker-images)
+    - [Building Docker images](#building-docker-images)
+    - [Tagging and pushing Docker images](#tagging-and-pushing-docker-images)
+- [Building everything](#building-everything)
+- [Release](#release)
+
+<!-- /TOC -->
+
+## Docker images
+
+### Building Docker images
+
+The `docker_build` target will build the Docker images provided by the 
+Barnabas project. You can build all Barnabas Docker images by calling 
+`make docker_build` from the root of the Barnabas repository. Or you can build 
+an individual Docker image by running `make docker_build` from the 
+subdirectories with their respective Dockerfiles - e.g. `kafka_base`, 
+`kafka_statefulsets` etc.
+
+The `docker_build` target will always build the images under the 
+`enmasseproject` organization. This is necessary in order to be able to reuse 
+the base image you might have just built without modifying all Dockerfiles.
+
+To configure the `docker_build` target you can set environment variables:
+* `DOCKER_ORG` should contain the Docker organization for tagging/pushing 
```

Comment:
The DOCKER_ORG environment variable is now ignored by `docker_build` (the paragraph above explains why). Could you please remove it? 

(Sorry, this is probably my mistake, not yours)